resources/views/localizations.blade.php

```diff
@@ -0,0 +1,135 @@
+                </div>
+                <div class="row">
+                    <div class="col s12">
+                        <blockquote>
+                            @lang('localizations.thank_translations'): <i>Szlovicsák Béla, Tóth Regina, Kovács Sára Kata, {{ implode(", ", $contributors) }}</i>!
+                        </blockquote>
+                    </div>
+                    @can('approve', App\LocalizationContribution::class)
+                    <div class="col s12">
+                        <a href="{{ route('localizations.admin') }}" class="btn right waves-effect"> @lang('localizations.manage_translations')</a>
+                    </div>
+                    @endcan
+                </div>
+                <script>
+                    function change_language(language){
+                    if(language=="hu"){
+                        document.getElementById("hu").classList.remove("hide");
+                        document.getElementById("en").classList.add("hide");
+                    }
+                    if(language=="en"){
+                        document.getElementById("en").classList.remove("hide");
+                        document.getElementById("hu").classList.add("hide");
+                    }
+                    //set textarea heights
+                    ($('textarea')).each(function(index, elem){
+                        elem.style.height = elem.scrollHeight+'px'; 
+                    });
+                }
+                </script>
+            </div>
+        </div>
+    </div>
+    @php
+    /* Require reference langugage files - except validation.php */
+    $reference_files = [
+        'en' => array_diff(scandir(base_path('resources/lang/en')), ['..', '.', 'validation.php']),
+        'hu' => array_diff(scandir(base_path('resources/lang/hu')), ['..', '.', 'validation.php'])
+    ];
+    @endphp
+    @if(App::getLocale() != 'hu' && App::getLocale() != 'en')
+    @foreach ($reference_files as $lang => $files)
+    <div id="{{ $lang }}">
+        @foreach ($files as $file)
+        @php
+        $fname = substr($file, 0, -4); //filename without .php, used as a part of the expression key
+        $expressions = require base_path('resources/lang/'.$lang.'/'.$file);
+        @endphp
+        <div class="col s12">
+            <div class="card">
+                <div class="card-content">
+                    <div class="card-title" id="{{ $fname }}">{{ $fname }}</div>
+                    @foreach ($expressions as $key => $value)
+                    {{-- Not shown if the expression have an ongoing, unapproved change --}}
+                    @php
+                    $duplicate = App\LocalizationContribution::where('language', App::getLocale())->where('key',  $fname.'.'.$key)->where('approved', false)->first();
+                    @endphp
+                    @if(is_string($value) && $duplicate == null)
+                    <form method="POST" action="{{ route('localizations.add') }}">
```

Comment:
This should be done through AJAX calls. Let's open an issue for that.

Reply:
I fixed it here, UX is much better.